.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run prettier && git add -A .
```

Comment:
This is little dangerous from my point of view. As you might want to commit only part of the file but you will get full file after it ... 

I will add some notes to the ticket (not pull request)